packages/identity/tests/index.test.ts

```diff
@@ -2,20 +2,20 @@ import { Identity } from "../src"
 
 describe("Identity", () => {
     const privateKeyText = "secret"
-    const privateKeyHexadecimal = "dd998334940df8931b76d899fdb189415f7ff4280599f03a7574725a166aad7d"
+    const privateKeyBuffer = Buffer.from(privateKeyText)
 
     describe("# Identity", () => {
         it("Should create an identity with a random secret (private key)", () => {
             const identity = new Identity()
 
-            expect(typeof identity.privateKey).toBe("string")
-            expect(identity.privateKey).toHaveLength(64)
+            expect(typeof identity.privateKey).toBe("object")
+            expect(identity.privateKey).toHaveLength(32)
             expect(typeof identity.secretScalar).toBe("bigint")
             expect(identity.publicKey).toHaveLength(2)
             expect(typeof identity.commitment).toBe("bigint")
         })
 
-        it("Should create deterministic identities from a secret text (private key)", () => {
+        it("Should create a deterministic identity from a secret text (private key)", () => {
```

Comment:
These first 3 tests aren't really checking for "deterministic" output to the extent I'd expect, since they aren't checking the value of the output (just it's type and length).
I suggest having a hard-coded "expected value" for the secretScalar, commitment, and public key to check explicitly.  In this PR you'd probably generate that test data by printing the result, so it may seem like it's not testing anything new.  However it'll fail in future if the output produced by the same input ever changes, which would be a compatibility-breaking bug, and good to catch.  You could also use this approach to ensure that text vs buffer input produce the same output when they're supposed to.